src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Component {
```

Comment:
According to the checklist, custom annotations should have a `@Target` annotation to specify where this annotation can be used. For example, if `@Component` is meant to be used on classes, you should add `@Target(ElementType.TYPE)`.

Reply:
According to the checklist, custom annotations should have a @Target annotation to specify where this annotation can be used. For example, if @Component is meant to be used on classes, you should add @Target(ElementType.TYPE).